civitai-downloader-v2/tests/unit/test_error_handling.py

```
@@ -0,0 +1,292 @@
+        UnifiedErrorHandler = handler_module.UnifiedErrorHandler
+        error_handler = UnifiedErrorHandler()
+        
+        async def test_async_operation():
+            # Mock async operation that fails
+            async def failing_operation():
+                await asyncio.sleep(0.1)
+                raise ConnectionError("Async operation failed")
+            
+            # Test async error handling
+            try:
+                result = await error_handler.handle_async_error(
```

Ruff (0.12.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)